reparse_test.go

```diff
@@ -0,0 +1,113 @@
+package winio
```

Comment:
Can you add a `//go:build windows` constraint to this file and see if that resolves the Linux CI problem?